langchain/src/chat_models/tests/chatbaiduwenxin.int.test.ts

```diff
@@ -0,0 +1,126 @@
+
+test.skip("Test ChatBaiduWenxin default model", async () => {
+  const chat = new ChatBaiduWenxin();
+  const message = new HumanChatMessage("Hello!");
+  const res = await chat.call([message]);
+  console.log({ res });
+});
+
+test.skip("Test ChatBaiduWenxin ERNIE-Bot", async () => {
+  const chat = new ChatBaiduWenxin({ modelName: "ERNIE-Bot" });
+  const message = new HumanChatMessage("Hello!");
+  const res = await chat.call([message]);
+  console.log({ res });
+});
+
+test.skip("Test ChatBaiduWenxin ERNIE-Bot with temperature", async () => {
+  const chat = new ChatBaiduWenxin({
+    modelName: "ERNIE-Bot",
+    temperature: 1,
+  });
+  const message = new HumanChatMessage("Hello!");
+  const res = await chat.call([message]);
+  console.log({ res });
+});
+
+test.skip("Test ChatBaiduWenxin ERNIE-Bot with topP", async () => {
+  const chat = new ChatBaiduWenxin({
+    modelName: "ERNIE-Bot",
+    topP: 1,
+  });
+  const message = new HumanChatMessage("Hello!");
+  const res = await chat.call([message]);
+  console.log({ res });
+});
+
+test.skip("Test ChatBaiduWenxin ERNIE-Bot with penaltyScore", async () => {
+  const chat = new ChatBaiduWenxin({
+    modelName: "ERNIE-Bot",
+    penaltyScore: 1,
+  });
+  const message = new HumanChatMessage("Hello!");
+  const res = await chat.call([message]);
+  console.log({ res });
+});
+
+test.skip("Test ChatBaiduWenxin ERNIE-Bot in streaming mode", async () => {
+  let nrNewTokens = 0;
+  let streamedCompletion = "";
+
+  const model = new ChatBaiduWenxin({
+    modelName: "ERNIE-Bot",
+    streaming: true,
+    callbacks: [
+      {
+        async handleLLMNewToken(token: string) {
+          nrNewTokens += 1;
+          streamedCompletion += token;
+        },
+      },
+    ],
+  });
+  const message = new HumanChatMessage("您好，请讲个长笑话");
+  const res = await model.call([message]);
+  console.log({ res });
+
+  expect(nrNewTokens > 0).toBe(true);
+  expect(res.text).toBe(streamedCompletion);
+});
+
+test.skip("Test ChatBaiduWenxin ERNIE-Bot-turbo", async () => {
+  const chat = new ChatBaiduWenxin({
+    modelName: "ERNIE-Bot-turbo",
+  });
+  const message = new HumanChatMessage("Hello!");
+  const res = await chat.call([message]);
+  console.log({ res });
+});
+
+test.skip("Test ChatBaiduWenxin ERNIE-Bot-turbo in streaming mode", async () => {
+  let nrNewTokens = 0;
+  let streamedCompletion = "";
+
+  const model = new ChatBaiduWenxin({
+    modelName: "ERNIE-Bot-turbo",
+    streaming: true,
+    callbacks: [
+      {
+        async handleLLMNewToken(token: string) {
+          nrNewTokens += 1;
+          streamedCompletion += token;
+        },
+      },
+    ],
+  });
+  const message = new HumanChatMessage("您好，请讲个长笑话");
+  const res = await model.call([message]);
+  console.log({ res });
+
+  expect(nrNewTokens > 0).toBe(true);
+  expect(res.text).toBe(streamedCompletion);
+});
+
+test.skip("Test ChatBaiduWenxin tokenUsage", async () => {
```

Comment:
This test case is always failed. In `_generate()` I can `console.log` correct token data. But I don't know why the data is not passed here correctly. 

Could you help me to figure it out? Thanks.

Reply:
Not every provider returns it - I would say just remove it for now and try adding it later since this PR has already taken a lot of back and forth

Reply:
Removed the test case about tokenUsage.